pkg/controller/clusters/status.go

```diff
@@ -214,11 +222,26 @@ func updateWorkersReadyCondition(cluster *anywherev1.Cluster, machineDeployments
 		totalReplicas += int(md.Status.Replicas)
 	}
 
+	totalInplace := 0
+	totalRolling := 0
+	for _, worker := range cluster.Spec.WorkerNodeGroupConfigurations {
+		if worker.UpgradeRolloutStrategy.Type == anywherev1.InPlaceStrategyType {
+			totalInplace++
+		} else {
+			totalRolling++
+		}
+	}
```

Comment:
Reiterating Taylor's comment, if we don't support hybrid, we might not need this whole chunk instead just use control plane strategy to figure the upgrade type and update the condition below